### PR TITLE
feat: added alert component page and css file

### PR DESCRIPTION
**Pull request by zabihshaik**
Added Alert Component Page and it's related CSS File. 
Imported the alert.css file into main.css.

Alert Page Screenshot:

Alert Page

Deploy Preview Link:
https://deploy-preview-2--neoz-ui.netlify.app/

### CODE REVIEW

**Review by nyyshaaa — APPROVED** [review]
Looks good to me!

File: styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.alert-container {
+    flex-direction: column;
```
Comment (yashpatel1797) [inline styles/alert.css:3]: 'display: flex' is not declared here. without that other flex, properties will not apply.

**Review by yashpatel1797 — APPROVED** [review]
looks good to merge.

File: styles/alert.css (hunk 1 above)
Comment (zabihshaik) [reply styles/alert.css:3]: Will do it. Thanks for pointing it out.

**Comment by zabihshaik** [comment]
Fixed the minor changes as per PR Review.

**Pull request merged**